apps/api/src/lib/circle-adapter.ts

```diff
@@ -98,31 +94,12 @@ function toBankAccountBase(
   }
 }
 
-function toCardStatus(
-  status: CircleCardStatus,
-  verification: CircleCardVerification
-): PaymentCardStatus {
-  let finalStatus
-  if (
-    status === CircleCardStatus.Failed ||
-    Object.values(CircleVerificationAVSFailureCode).includes(
-      verification.avs
-    ) ||
-    verification.cvv === CircleVerificationCvvStatus.Fail
-  ) {
-    finalStatus = PaymentCardStatus.Failed
-  } else if (
-    status === CircleCardStatus.Complete &&
-    Object.values(CircleVerificationAVSSuccessCode).includes(
-      verification.avs
-    ) &&
-    verification.cvv === CircleVerificationCvvStatus.Pass
-  ) {
-    finalStatus = PaymentCardStatus.Complete
-  } else {
-    finalStatus = PaymentCardStatus.Pending
-  }
-  return finalStatus
+function toCardStatus(status: CircleCardStatus): PaymentCardStatus {
+  return {
+    [CircleCardStatus.Complete]: PaymentCardStatus.Complete,
+    [CircleCardStatus.Failed]: PaymentCardStatus.Failed,
+    [CircleCardStatus.Pending]: PaymentCardStatus.Pending,
+  }[status]
```

Comment:
We don't need to have these conditionals any longer, because Circle will resolve itself based on the cvv, 3DS, and avs checks already.